tensorflow_federated/python/core/impl/executors/federating_executor.py

```diff
@@ -486,6 +489,131 @@ def validate_executor_placements(cls, executor_placements):
                   'Unsupported cardinality for placement "{}": {}.'.format(
                       pl, pl_cardinality))
 
+  async def _trusted_aggregator_generate_keys(self):
+    
+    @computations.tf_computation()
+    def generate_keys():
+      pk, sk = easy_box.gen_keypair()
+      return pk.raw, sk.raw
+
+    fn_type = generate_keys.type_signature
+    fn = generate_keys._computation_proto
+
+    aggregator_ex = self._get_child_executors(placement_literals.AGGREGATOR,
+                                              index=0)
+
+    result = await aggregator_ex.create_call(
+        await aggregator_ex.create_value(fn, fn_type)
+    )
+
+    pk = await aggregator_ex.create_selection(result, 0)
```

Comment:
you could use `asyncio.gather` here instead to run these concurrently; same for two of the three below

Reply:
Good point, fixed!